frontend/kubecloud/src/components/dashboard/SshKeysCard.vue

```diff
@@ -95,47 +94,49 @@
 </template>
 
 <script setup lang="ts">
-import { ref, onMounted, nextTick } from 'vue'
+import { ref, onMounted, nextTick, computed } from 'vue'
 import { userService, type SshKey, type AddSshKeyRequest } from '../../utils/userService'
 
 const sshKeys = ref<SshKey[]>([])
 const addDialog = ref(false)
 const newKey = ref<AddSshKeyRequest>({ name: '', public_key: '' })
-const addError = ref('')
-const nameError = ref('')
-const keyError = ref('')
 const nameField = ref()
-const copySuccess = ref('')
+const lastCopiedId = ref<number | null>(null)
+
+const isValidSshKey = (key: string): boolean =>
+  /^(ssh-(rsa|ed25519|dss|ecdsa)|ecdsa-sha2-nistp\d+|sk-ecdsa-sha2-nistp\d+|sk-ssh-ed25519) [A-Za-z0-9+/=]+( [^@\s]+@[^@\s]+)?$/.test(key.trim())
+
+const isDuplicateKey = computed(() => {
+  const val = newKey.value.public_key.trim()
+  return !!val && sshKeys.value.some(k => k.public_key.trim() === val)
+})
+
+const nameError = computed(() => !newKey.value.name ? 'Key name is required.' : '')
```

Comment:
name must be unique as well 

<img width="624" height="89" alt="image" src="https://github.com/user-attachments/assets/b775999a-4290-4001-956c-705b3a65dfec" />